src/main/java/com/google/sps/data/ScheduledInterviewDao.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/**
+ * ScheduledInterviewDao includes the basic methods anything managing ScheduledInterview entities
+ * must support.
+ */
+public interface ScheduledInterviewDao {
+  // Returns a ScheduledInterview object from storage
+  public ScheduledInterview get() throws com.google.appengine.api.datastore.EntityNotFoundException;
```

Comment:
I think we discussed yesterday that you should wrap the datastore-specific exception objects in your own class. Here are some instructions on doing that: https://www.baeldung.com/java-new-custom-exception. Your custom NotFound exception should probably take a com.google.appengine.api.datastore.EntityNotFoundException as a constructor argument.